src/main/java/com/eszop/paymentservice/controller/PaymentController.java

```diff
@@ -0,0 +1,52 @@
+package com.eszop.paymentservice.controller;
+
+import com.eszop.paymentservice.dto.PaymentDTO;
+import com.eszop.paymentservice.entity.Payment;
+import com.eszop.paymentservice.repository.PaymentRepository;
+import org.modelmapper.ModelMapper;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/payment")
+public class PaymentController {
+    private final PaymentRepository repository;
```

Comment:
raczej robi się jeszcze warstwę serwisową gdzie możamy dany payment obsłużyć i tam bezpośrednio wołamy repository którym robimy akcję z bazą danych,
ja bym może wystawił jeszcze jakieś endpointy typu /payments/{id} do kreacji (to mamy)
/payments/{id}/in-progress żeby np była jakaś stronka i tam że już przechodzimy do płatności
/payments/{id}/finished i to samo dla canceled żeby były tak na prawdę jakieś eventy na które możemy zareagować, w świecie idealnym bym jeszcze przy GET /payments/{id} zrobił to HATEOS owo trochę i podawał jakie mamy aktualnie możliwe akcje dla paymentu, np jak jest w in progres to że możemy dac cancell albo finish, wtedy po stronie klienta łatwo jest ogarnac np jakie butony pokazac koncowemu urzyszkodnikowi